Objects/unicodeobject.c

```diff
@@ -13080,6 +13108,7 @@ unicode_endswith_impl(PyObject *self, PyObject *subobj, Py_ssize_t start,
     return PyBool_FromLong(result);
 }
 
+
```

Comment:
```suggestion
```

It appears that you decided to have 2 lines here whereas the rest of the file seems to have 1 line.

Reply:
My plan is 50 years is to fix the C style to have 2 lines between functions. Honestly, it's way more readable. I'm doing that for 10 years in various C files :-)

Reply:
> Maybe I can suggest you to make a follow-up PR where you would add two lines everywhere in the file then

We usually don't accept changes which are only coding style. It's a bad practice, it makes backporting way more complicated. There was a draft PEP to use formatting tools, but it stayed like a draft. So for now, I'm trying to do minor cleanup "around" my changes, not further.

Reply:
I once made a large change that added empty lines between methods in Tkinter, because the old code did not have empty lines between methods and was difficult to read. It did not spoil history and did not complicate backporting, because it did not modified existing lines, just added empty lines.

If there is similar issue in other files, I think it is not a problem to fix empty lines.

But I usually prefer to use a single empty line between functions/methods and larger span between more different parts of code. I think this is mentioned in PEP 7/8. Automatic formatters cannot understand semantic difference.